### ISSUE TITLE
gl2faststructure does not work with SilicoDArT data

### ISSUE THREAD

**Issue by niWdooG** [green-striped-gecko/dartR#86]
Hello,

I am trying to convert my SilicoDArT data to the .str format using gl2faststructure in dartR v. 1.5.5:

**gl2faststructure(gs_file,outfile=file.path("..", "filename.fs"), probar = TRUE)**,

but getting the following error: 

**Fatal Error: Detected Presence/Absence (SilicoDArT) data. Please provide a SNP dataset**.

Sincerely,
Evgenii

**Comment by ollybolly** [comment]
Hi Evengii,

Bernd and Arthur may correct me, but I believe this function is not designed for silicondart data. I recall that structure also is designed for use with codominant data.
Cheers,
Ollu

—
Leader, CSIRO Environomics Future Science Platform
www.csiro.au/environomics

Sent from my phone


________________________________

**Comment by niWdooG** [comment]
Hi Ollu,

The original Structure do work with dominant markers. 

Recently I have analysed Silicodart (SD) and SNPs markers using fastStructure (FS). To be able to run FS with SD data I prepared my input file in a diploid fashion:

Sample1       1       1      -9	1	0	
Sample1       1	      1      -9	1	0		
Sample2       0	      1       0	1	1	
Sample2       0	      1       0	1      1

FS outputs showed almost the same results. However, the SD markers revealed one additional cluster that better explains my data.

Sincerely,
Evgenii
